### PR TITLE
Add new tools: Clair, Snyk, Grype

### PR DESCRIPTION
**Pull request by reynico**

### CODE REVIEW

**Comment by reynico** [comment]
wrong upstream heh

File: Dockerfile [hunk 1]
```diff
@@ -155,13 +155,29 @@ RUN wget -qO - https://aquasecurity.github.io/trivy-repo/deb/public.key | gpg --
...
+RUN ARCH=$(dpkg --print-architecture) \
+    && if [ "$ARCH" = "amd64" ]; then \
+    sudo wget -qO /usr/local/bin/snyk https://github.com/snyk/cli/releases/download/v1.1293.1/snyk-linux; \
+    elif [ "$ARCH" = "arm64" ]; then \
+    sudo wget -qO /usr/local/bin/snyk https://github.com/snyk/cli/releases/download/v1.1293.1/snyk-linux-arm64; \
+    fi \
+    && sudo chmod +x /usr/local/bin/snyk
+
+# Install Grype
+RUN ARCH=$(dpkg --print-architecture) \
```
Comment (mattaereal) [inline Dockerfile]:
```suggestion
RUN curl -sSfL https://raw.githubusercontent.com/anchore/grype/main/install.sh | sudo sh -s -- -b /usr/local/bin
```

File: Dockerfile [hunk 2]
```diff
@@ -155,13 +155,29 @@ RUN wget -qO - https://aquasecurity.github.io/trivy-repo/deb/public.key | gpg --
...
+    && wget -q https://github.com/anchore/grype/releases/download/v0.81.0/grype_0.81.0_linux_$ARCH.deb \
```
Comment (mattaereal) [inline Dockerfile]:
```suggestion
```

File: Dockerfile [hunk 3]
```diff
@@ -155,13 +155,29 @@ RUN wget -qO - https://aquasecurity.github.io/trivy-repo/deb/public.key | gpg --
...
+    && sudo dpkg -i grype_0.81.0_linux_$ARCH.deb \
```
Comment (mattaereal) [inline Dockerfile]:
```suggestion
```

File: Dockerfile [hunk 4]
```diff
@@ -155,13 +155,29 @@ RUN wget -qO - https://aquasecurity.github.io/trivy-repo/deb/public.key | gpg --
...
+    && rm grype_0.81.0_linux_$ARCH.deb
```
Comment (mattaereal) [inline Dockerfile]:
```suggestion
```

File: Dockerfile [hunk 5]
```diff
@@ -155,13 +155,29 @@ RUN wget -qO - https://aquasecurity.github.io/trivy-repo/deb/public.key | gpg --
...
+RUN wget -qO - https://github.com/checkmarx/2ms/releases/latest/download/linux-amd64.zip | \
+    funzip - | sudo tee /usr/local/bin/2ms > /dev/null \
+    && sudo chmod +x /usr/local/bin/2ms
+
+# Install clair
+RUN sudo wget -qO /usr/local/bin/clair https://github.com/quay/clair/releases/download/v4.7.4/clairctl-linux-$(dpkg --print-architecture) \
+    && sudo chmod +x /usr/local/bin/clair
+
+# Install snyk
+RUN ARCH=$(dpkg --print-architecture) \
```
Comment (mattaereal) [inline Dockerfile]:
```suggestion
RUN pnpm install -g snyk
```
I'd suggest installing it directly from the repo, and delete the rest

**Review by mattaereal — CHANGES_REQUESTED** [review]
I installed grype and snyk differently. Tried to suggest small changes via gh's UI but horrible experience. 

A few PRs after this I successfully installed them both las follows:
`RUN pnpm install -g snyk`
`RUN curl -sSfL https://raw.githubusercontent.com/anchore/grype/main/install.sh | sudo sh -s -- -b /usr/local/bin`

File: Dockerfile (hunk 4 above)
Comment (reynico) [reply Dockerfile:178]: y that?

**Comment by reynico** [comment]
> I installed grype and snyk differently. Tried to suggest small changes via gh's UI but horrible experience.
> 
> A few PRs after this I successfully installed them both las follows: `RUN pnpm install -g snyk` `RUN curl -sSfL https://raw.githubusercontent.com/anchore/grype/main/install.sh | sudo sh -s -- -b /usr/local/bin`

gotcha! just addressed the comments and committed a last fix to the suggestions. CI is passing as expected so I think this is good to go

**Review by mattaereal — APPROVED** [review]

**Pull request merged**